data/json/items/chemicals_and_resources.json

```diff
@@ -540,31 +540,73 @@
     "count" : 10
    },
    {
-    "type" : "AMMO",
+    "type" : "COMESTIBLE",
```

Comment:
Look at the effects, though: low fun, negative health and quench etc.
It may be considered a food product IRL, but in game it has more negative effects than meth (while that's more of an issue with meth, it should be considered here too).

Plus, I'm pretty sure drinking pure, undiluted ethanol IRL is at best a party trick. Especially considering that you can just dilute it with water.

While there could be a rare situation in game where a desperate alcohol addict wants to consume pure ethanol, those will be really, really rare and far outnumbered by accidental consumption.

Bleach is also consumable, but this is a result of it not being updated up to standards rather than a conscious decision to keep it that way.